test/unit/spec/EnrollChoices_spec.js

```diff
@@ -127,6 +127,36 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
       return setup(res);
     }
 
+    function setupMultipleU2FEnrollments(status, enrollment, policyEnrollment) {
+      var res = deepClone(resAllFactors);
+      res.response._embedded.factors[10].status = status;
+      res.response._embedded.factors[10].enrollment = enrollment;
+      res.response._embedded.factors[10].policy.enrollment = policyEnrollment;
+      return setup(res);
+    }
+    
+    function setupMultipleU2FEnrollmentsEnrolledOptional(policyEnrollment) {
```

Comment:
would it make more sense to name this according to the values we use? Like `setupMultipleU2FWithActiveStatusAndOptionalEnrollment`
Same for the other functions below. It might be easier to understand what the exactly the function does